.golangci.yml

```diff
@@ -119,3 +119,12 @@ issues:
     - linters: [revive]
       text: "^if-return: "
       path: error_writer.go
+    # We want to set http.Server's logger
+    - linters: [forbidigo]
+      path: internal/memhttp
+    # We want to set http.Server's logger
+    - linters: [forbidigo]
+      path: internal/memhttp/memhttptest
```

Comment:
I'm pretty sure `path` here is a regex, which means the above rule should already match this path. So I think this one might be redundant and can be removed.